projects/observability/src/shared/components/table/data-cell/entity/entity-table-cell-renderer-util.ts

```diff
@@ -27,3 +30,22 @@ export const parseEntityFromTableRow = (cell: Entity | undefined, row: Dictionar
 };
 
 const isInteraction = (neighbor: unknown): neighbor is Interaction => typeof neighbor === 'object';
+
+export const isInactiveEntity = (row: TableRow): boolean | undefined => {
+  const maxEndTimeAggregation = row['max(endTime)']; // Ew.
```

Comment:
This seems... ripe for improvement. Unless we change the api to pass this info in (which wouldn't be a bad idea but likely won't happen immediately), a better heuristic for an inactive entity that avoids looking at any particular field which may or may not be requested/exist on all entities would be to check:
1. The entity includes at least one metric AND
2. All metrics fetched are undefined

Reply:
Yeah, this is really really ugly. I felt better putting it in this file which already holds some ugly Entity handling code. At least it is all isolated.

Talked with the backend folks about which heuristic to use here. Checking that `max(endTime)` is a valid number is the most solid way to know for sure.

Reply:
> Talked with the backend folks about which heuristic to use here. Checking that max(endTime) is a valid number is the most solid way to know for sure.

See above - max(endTime) is a specific subset of the general heuristic I described which allows us to remove any coupling to a specific usage. No metric will be available for an inactive entity, so it just limits us to pick one to check.